test/integration/test_http.py

```diff
@@ -73,5 +73,196 @@ def test_basicTxn(self):
         print(result)
         self.assertEqual(result["status"]["code"], 200);
 
+    # Helper method to populate and send request with default values
```

Comment:
I find the test difficult to read as all it does is making calls to a single function with 11 fields, most of which with default values that aren't very intuitive. There are also a lot of repeated string constants that I can't tell if you meant them to be what they are or you have a copy/paste error

Why not create helpers with API similar to a client library instead? We can actually develop this  as an http client for python.

```python
class QueryResponse:

class Txn:
  def __init__(self, address): pass
  # return (status, record)
  def read(self, record): pass
  # return (status)
  def write(self, record): pass
  # return (status, QueryResponseObj)
  def scan(self, query): pass

class Client:
  # return (status, txnObj)
  def beginTxn(self): pass

  # return (status)
  def createCollection(self, collname, ranges,....): pass

  # return (status, schemaObj)
  def createSchema(self, collname, schemaName, ....): pass
```

then a test would be, e.g.
``` python
def test_conflict(self):
    coll1 = 'testCollection01'
    schema1 = 'testSchema01'
    client = Client(args.http)
    txn1 = client.beginTxn()
    txn2 = client.beginTxn()

    status = txn1.write(coll1, schema1, {'pkey': 'partitionKey1',
               'rkey': 'rangeKey1', 'field1': 'abcd', 'field2': 'efgh'})
    self.assertEqual(status.code, 201)
    status = txn2.write(coll1, schema1, {'pkey': 'partitionKey1',
               'rkey': 'rangeKey1', 'field1': 'abcd', 'field2': 'efgh'})
    self.assertEqual(status.code, 409)

```

Reply:
Working on it.

Reply:
Added initial version of python client in `skvclient.py`. It's mainly to simplify calling read/write api without passing all  parameters as tests call apis with same parameter except one parameter different.